packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -71,6 +71,7 @@ export function RepositoryMixin<T extends Class<any>>(superClass: T) {
         name,
         namespace: 'repositories',
         type: 'repository',
+        defaultScope: BindingScope.TRANSIENT,
```

Comment:
Do we have a test to verify this behavior?

Also since TRANSIENT is the default scope anyways, is it necessary to explicitly configure it here again? I am fine either way, just asking.